static/sass/styles.scss

```diff
@@ -117,3 +117,9 @@
     overflow: scroll;
   }
 }
+
+.p-list--divided.is-contact {
+  li {
+    padding-bottom: 0;
+  }
+}
```

Comment:
Personally, I would add `u-no-padding--bottom` to the li's instead of a new class.